### PR TITLE
Multiple browser contexts

### LINKED ISSUE #12: API to handle multiple browser contexts
It would be good to have a way to manage multiple browsers contexts, either defined at the start of the crawl (in settings, for instance) or created/destroyed during the crawl.

/cc @kalessin

### PR DESCRIPTION
**Pull request by elacuesta**
Closes #12

Tasks:
- [x] Multiple contexts at startup
- [x] Create new contexts during a crawl
- [x] Example spider
- [x] Settings docs
- [x] Usage docs
- [x] Tests
- [x] Deprecate `PLAYWRIGHT_CONTEXT_ARGS` in favour of `PLAYWRIGHT_CONTEXTS`

### CODE REVIEW

**Comment by elacuesta** [comment]
@kalessin could you take a look at this? Thanks!

File: README.md [hunk 1]
```diff
@@ -176,6 +164,105 @@ class AwesomeSpiderWithPage(scrapy.Spider):
...
+            "ignore_https_errors": True,
+            "proxy": {
+                "server": "http://myproxy.com:3128",
+                "username": "user",
+                "password": "pass",
+            },
+        },
+    },
+)
+```
```
Comment (Gallaecio) [inline README.md:206]: What happens if a context already exists with the same name?

Comment (elacuesta) [reply README.md:206]: The existing context is used in that case, and `playwright_context_kwargs` are ignored.

Comment (elacuesta) [reply README.md:207]: I just added a note to the readme about ignoring the kwargs.

**Review by Gallaecio — APPROVED** [review]

**Pull request merged**